### PR TITLE
Fix Anomaly Detection System Indices Deletion in Cypress Tests

### PR DESCRIPTION
**Pull request by kaituo**
### Description

This commit addresses an issue where the deletion of Anomaly Detection (AD) system indices was failing in Cypress tests due to lack of super admin privileges. Instead of attempting to delete the indices as super admin, which isn't feasible in the Cypress environment, we now employ an API-based approach to delete the content of these AD indices.

Testing Performed:
1. Executed all tests in a security-enabled cluster confirming the successful deletion of AD system indices content.

### Issues Resolved

https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/714

### Check List

- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/utils/commands.js [hunk 1]
```diff
@@ -108,7 +108,45 @@ Cypress.Commands.add('deleteAllIndices', () => {
...
+    method: 'POST',
+    url: `${Cypress.env(
+      'openSearchUrl'
+    )}/_plugins/_anomaly_detection/detectors/_search`,
+    failOnStatusCode: false,
+    body: { query: { match_all: {} } },
+  }).then((response) => {
+    if (response.status === 200) {
+      for (let hit of response.body.hits.hits) {
+        console.log(hit);
```
Comment (ohltyler) [inline cypress/utils/commands.js:131]: can remove

**Review by ohltyler — APPROVED** [review]
LGTM! Just one leftover log line

File: cypress/utils/commands.js (hunk 1 above)
Comment (kaituo) [reply cypress/utils/commands.js]: removed

**Review by kavilla — APPROVED** [review]

**Comment by kaituo** [comment]
> does this need a backport to 2.8?
@kavilla yeah, will backport to 2.8 and 2.x.

**Comment by kavilla** [comment]
> > does this need a backport to 2.8?
> 
> @kavilla yeah, will backport to 2.8 and 2.x.

@kaituo done!

**Comment by kaituo** [comment]
@kavilla Thanks a ton :)

**Pull request merged**